src/Illuminate/Cache/ApcStore.php

```diff
@@ -49,6 +49,24 @@ public function get($key)
     }
 
     /**
+     * Retrieve multiple items from the cache by key,
```

Comment:
Also, note the changes I made to the param annotation in that snippet. Please do that too.